pkg/local/vmis.go

```diff
@@ -125,30 +125,72 @@ func newVMIs(binding *v1beta1v8o.VerrazzanoBinding, verrazzanoUri string, enable
 					Enabled:             true,
 					Storage:             storageOption,
 					DashboardsConfigMap: util.GetVmiNameForBinding(binding.Name) + "-dashboards",
+					Resources: vmov1.Resources{
+						RequestMemory: getGrafanaRequestMemory(),
+					},
 				},
 				IngressTargetDNSName: fmt.Sprintf("verrazzano-ingress.%s", verrazzanoUri),
 				Prometheus: vmov1.Prometheus{
 					Enabled: true,
 					Storage: storageOption,
+					Resources: vmov1.Resources{
+						RequestMemory: getPrometheusRequestMemory(),
+					},
 				},
 				Elasticsearch: vmov1.Elasticsearch{
 					Enabled: true,
 					Storage: storageOption,
 					IngestNode: vmov1.ElasticsearchNode{
 						Replicas: 1,
+						Resources: vmov1.Resources{
+							RequestMemory: getElasticsearchIngestNodeRequestMemory(),
+						},
 					},
 					MasterNode: vmov1.ElasticsearchNode{
-						Replicas: 1,
+						Replicas: 3,
+						Resources: vmov1.Resources{
+							RequestMemory: getElasticsearchMasterNodeRequestMemory(),
+						},
 					},
 					DataNode: vmov1.ElasticsearchNode{
 						Replicas: 2,
+						Resources: vmov1.Resources{
+							RequestMemory: getElasticsearchDataNodeRequestMemory(),
+						},
 					},
 				},
 				Kibana: vmov1.Kibana{
 					Enabled: true,
+					Resources: vmov1.Resources{
+						RequestMemory: getKibanaRequestMemory(),
+					},
 				},
 				ServiceType: "ClusterIP",
 			},
 		},
 	}
 }
+
+func getElasticsearchMasterNodeRequestMemory() string {
+	return "1.9Gi"
+}
+
+func getElasticsearchIngestNodeRequestMemory() string {
```

Comment:
Where are you getting this number?

Reply:
reduced to 2.5Gi, which is real usage + 10%